src/Api/lib/ApiV3.ts

```diff
@@ -179,9 +179,18 @@ export namespace ApiV3 {
                 }
               );
             } else {
+              const text = await response.text();
+              console.log(text);
+              let data = null;
+              try {
+                const json = JSON.parse(text);
```

Comment:
I think it makes sense to check if the response is in json format before attempting to parse. something similar to https://github.com/ZaiusInc/opti-cli/pull/69/files#diff-1840da001b419f908b0a37b59bd80d01bcfaf3dfc66ee17cdd1ecfc775a5ecd0R23